control-plane/roles/monitoring/templates/grafana_ingress.yaml

```diff
@@ -0,0 +1,27 @@
+---
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: monitoring-ingress-grafana
+  annotations:
+    cert-manager.io/cluster-issuer: letsencrypt-prod
```

Comment:
This should be factored out into variables for the ingress.

Reply:
https://github.com/metal-stack/metal-roles/commit/7c42e882681d058ef2177e01a7f590e8d573a99f